DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgDocumentContentRepository.cs

```diff
@@ -41,14 +40,13 @@ public int Add(DocumentContentModel entity)
         _logger.LogDebug("Adding DocumentContent with id {DocumentId} to database", entity.DocumentId);
         _logger.LogTrace("DocumentContent: {DocumentContent}", entity);
         using IDbConnection con = _connectionFactory.CreateConnection();
-        return con.Execute(
-            "insert into document_contents(documents_id, content)" +
-            " values (@DocumentId, @Content)",
-            new
-            {
-                entity.DocumentId,
-                entity.Content
-            });
+        return con.Execute("insert into document_contents(documents_id, content)" +
+                        " values (@DocumentId, @Content)",
```

Comment:
```suggestion
        return con.Execute(
                        "insert into document_contents(documents_id, content)" +
                        " values (@DocumentId, @Content)",
```

Reply:
men er det pænere? hvad med?
```suggestion
        return con.Execute("insert into document_contents(documents_id, content)" +
                                       " values (@DocumentId, @Content)",
```

Reply:
Det er mere consistent med resten af kodebasen.

Reply:
okay